CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to the Mila Docs
+
+Thank you for your interest into making a better documentation for all at Mila. Here are some guidelines to help bring your contributions to life.
+
+## What could be included
+
+* Mila cluster usage
+* Compute Canada cluster usage
+* Job management tips / tricks
+* Research good practices
+* Software development good practices
+* Useful tools
+
+## Issues / Pull Requests
+
+### Issues
+
+Issues can be used to report any error in the documentation, missing or unclear sections, broken tools or other suggestions to improve the overall documentation.
+
+### Pull Requests
+
+PRs are welcome! Keep the lines width to 80 characters or less and reference the related issues like this:
+
+```
+Resolves: #123
+See also: #456, #789
+```
+
+You can attempt to build the docs yourself to see if the formating is right:
+
+```console
+python3 -m pip install -r docs/requirements.txt
+sphinx-build -b html docs/ docs/_build/
+```
+
+This will produce the html version of the documentation which you can navigate by opening `docs/_build/index.html`.
+
+If you have any touble building the docs, don't hesitate to open an issue to request help or simply provide the content you would like to add in markdown if that is simpler for you.

```

Comment:
We value the contents of your contributions over the appearance or functionality of your pull request. Even if you don't know how to write the proper markup in either RestructuredText or Markdown your contributions can be greatly valuable. If you'd like to add markup but are uncertain on how to do so, we encourage you to add a TODO comment in line with the document explaining what type of markup you would envision. 

For instance : 

`.. todo Add link to the given site.

Reply:
I like the first part:

> We value the contents of your contributions over the appearance or functionality of your pull request. Even if you don't know how to write the proper markup in either RestructuredText or Markdown your contributions can be greatly valuable.

But I don't understand the objective of the second part. Is it to add support to other markup languages? I'm not sure how the TODO comment would concretize either. If it is to request support for a new markup language, it seams an issue would be better for that rather than mixing it in a PR?

Reply:
True, that is a much better idea. Thx !